test/units/cloneNode.ts

```diff
@@ -0,0 +1,31 @@
+import { DOMParser } from "../../deno-dom-wasm.ts";
+import { Element } from "../../src/dom/element.ts";
+import { Node } from "../../src/dom/node.ts";
```

Comment:
Sorry, I missed this, but please import both `Element` and `Node` from `deno-dom-wasm.ts` and not from anything in `src/`